code/game/turfs/simulated/wall_attacks.dm

```diff
@@ -339,26 +339,31 @@
 		return TRUE
 
 	// Attack the wall with items
-	if(istype(W,/obj/item/rcd) || !istype(W, /obj/item/chems))
+	if(istype(W,/obj/item/rcd) || istype(W, /obj/item/chems))
 		return
 	if(!W.force)
 		return
 	if(isliving(user))
 		var/mob/living/L = user
 		if(L.a_intent == I_HELP)
 			return
-	var/dam_threshhold = material.integrity
-	if(reinf_material)
-		dam_threshhold = ceil(max(dam_threshhold,reinf_material.integrity)/2)
-	var/dam_prob = min(100,material.hardness*1.5)
-	if(dam_prob < 100 && W.force > (dam_threshhold/10))
-		playsound(src, 'sound/effects/metalhit.ogg', 50, 1)
-		if(!prob(dam_prob))
-			visible_message("<span class='danger'>\The [user] attacks \the [src] with \the [W] and it [material.destruction_desc]!</span>")
-			dismantle_wall(1)
-		else
-			visible_message("<span class='danger'>\The [user] attacks \the [src] with \the [W]!</span>")
-	else
-		visible_message("<span class='danger'>\The [user] attacks \the [src] with \the [W], but it bounces off!</span>")
+	
+	user.do_attack_animation(src)
+	if(W.force < 5)
+		visible_message(SPAN_DANGER("\The [user] attacks \the [src] with \the [W], but it has no effect!"))
```

Comment:
imo `pick(W.attack_verb)`, attacks sounds bit too like it's a log line for OOC or something